apps/address-service/domains/common/utils/services/search/plugins/SearchByFiasId.js

```diff
@@ -0,0 +1,67 @@
+const { AbstractSearchPlugin } = require('./AbstractSearchPlugin')
+const { generateAddressKey } = require('@address-service/domains/common/utils/addressKeyUtils')
+const get = require('lodash/get')
```

Comment:
we se to sort imports

Reply:
As I know we have no tools to sort imports at backend scripts.

Reply:
But ok, sorted manually alphabetically.